

try:
    open('()', 'r')

except Exception as e:
    print('ddd', e)
